packages/block-editor/src/components/block-tools/style.scss

```
@@ -283,6 +283,11 @@
 		background: none;
 		border: none;
 	}
+
+	// Make the spacing consistent between controls.
+	.components-button {
+		height: 40px;
```

Any reason this shouldn't use $button-size-next-default-40px?

---

```suggestion
		height: $button-size-next-default-40px;
```

@richtabor Not as far as I know. However, when I checked out this PR the implementation doesn't quite work. 

I would look to fix this, but I actually find it _really_ hard to notice the differences/inconsistencies in spacing. My brain plays tricks.

Moreover, sometimes even when the spacing between items is "correct" it still looks visually "off" because of the relative space occupied by the icons in use. I wonder whether we'll need a default spacing and then custom / specific spacing on a case by case basis?